Add Kibana Chart

wilkers-steve opened the pull request:
Adds a basic Kibana Chart for use with Elasticsearch and Fluentd

wilkers-steve committed:
Add Kibana Chart

Adds a basic Kibana Chart for use with Elasticsearch and Fluentd

larryrensing commented on kibana/templates/deployment.yaml:
```diff
@@ -0,0 +1,55 @@
+      k8s-app: kibana-logging
+  template:
+    metadata:
+      labels:
```
define an update strategy in here

larryrensing commented on kibana/templates/service.yaml:
```diff
@@ -0,0 +1,31 @@
+    protocol: TCP
+    targetPort: {{ .Values.network.target_port }}
+  selector:
+    k8s-app: kibana-logging
```
perhaps add a way for someone to expose a nodeport?

wilkers-steve committed:
Refactor Kibana Chart

This brings Kibana more in line with the other charts in
Openstack-Helm. It leverages the Kolla kibana image, and allows
for overrides for a majority of the kibana configuration options
in the kibana configuration file

alanmeadows commented:
Steve, should this be closed as it is now destined for github.com/openstack?

wilkers-steve commented:
@alanmeadows affirmative
